src/components/ClientDashboard/Vehicles/EachVehicle.js

```diff
@@ -1,13 +1,66 @@
 import React from "react";
 import { MDBIcon } from "mdbreact";
```

Comment:
Thank you for the styling updates, this page looks great now! The image "alt text" could be improved.